frappe/__init__.py

```diff
@@ -618,8 +618,6 @@ def wrapper_fn(*args, **kwargs):
 
 			try:
 				retval = fn(*args, **get_newargs(fn, kwargs))
-			except:
-				raise
```

Comment:
Intentional?

This will catch and ignore all exceptions.

Reply:
Ignore. It's fine as long as `finally` doesn't `return`, `continue` or `break`.

https://docs.python.org/3/reference/compound_stmts.html#the-try-statement